README.md

```diff
@@ -6,17 +6,21 @@
                -> CONTENTS/{MEDIA}
                -> CONTENTS/{CONTAINER}
 
-        {PROJECT} -> CONTENTS/{CONTAINER}
+        {PROJECT} -> {CHAPTER}
                   -> CONTENTS/{MEDIA}
+                  -> CONTENTS/{CONTAINER}
 
-            {CONTAINER} -> {MEDIA}
+            {CHAPTER} -> {CONTAINER}
+                      -> {MEDIA}
```

Comment:
These should be prefixed with CONTENTS like with project

Reply:
Do you mean it should be chapter -> CONTENTS -> container/media?